dotnet-desktop-guide/framework/wpf/advanced/how-to-alter-the-typography-of-text.md

```diff
@@ -1,7 +1,8 @@
 ---
-title: How to Alter the Typography of Text
+title: "How to Alter the Typography of Text"
```

Comment:
I just noticed that the titles aren't supposed to be title case anymore. So unless it's a proper noun or code type, it should just simply be
```suggestion
title: "How to alter the typography of text"
```

Can you alter the other files you've changed?

Reply:
@adegeo Thank you for the suggestion! I have updated the content for the same. Please do have a look. :)